package.json

```diff
@@ -32,7 +32,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "forta-agent": "^0.1.48"
+    "forta-agent": "^0.1.48",
+    "forta-agent-tools": "^3.2.12"
```

Comment:
Since it is not used in production, `forta-agent-tools` could be put in `devDependencies`.